perftests/src/main/java/org/apache/qpid/disttest/controller/config/ConfigReader.java

```diff
@@ -37,10 +37,10 @@
 
     public Config getConfigFromFile(String fileName) throws IOException
     {
-        Reader reader = getConfigReader(fileName);
-
-        Config config = readConfig(reader);
-        return config;
+        try (Reader reader = getConfigReader(fileName))
```

Comment:
The reader is closed in  `ObjectMapper#read...`. Thus, the change is unnecessary...